java/ql/src/Security/CWE/CWE-209/SensitiveDataExposureThroughErrorMessage.ql

```diff
@@ -0,0 +1,22 @@
+/**
+ * @name Information exposure through an error message
+ * @description Information from an error message propagates to an external user.
+ *              Error messages can unintentionally reveal implementation details
+ *              that are useful to an attacker for developing a subsequent exploit.
+ * @kind problem
+ * @problem.severity error
+ * @security-severity 5.4
+ * @precision high
+ * @id java/error-message-exposure
+ * @tags security
+ *       external/cwe/cwe-209
+ */
+
+import java
+import semmle.code.java.dataflow.DataFlow
+import semmle.code.java.security.SensitiveDataExposureThroughErrorMessageQuery
+
+from Expr externalExpr, Expr errorInformation
+where
+  getMessageFlowsExternally(DataFlow::exprNode(externalExpr), DataFlow::exprNode(errorInformation))
```

Comment:
The current changes does not include checking if the error message is logged or printed out: only that it is exposed in HTTP bodies. 

If expanding this query with that check have interest then i can implement this: But likely in another issue and PR: Just to get this PR closed.